src/CMake/FindOSMesa.cmake

```diff
@@ -206,6 +206,7 @@ if (VISIT_OSMESA_DIR)
         endif()
     endif(VISIT_LLVM_DIR)
 
+    list(APPEND OSMESA_LIBRARIES ${LLVM_LIBRARY} ${DL_LIB})
```

Comment:
LLVM_LIBRARY already appended to OS_MESA_LIBRARIES, why do it again?

Reply:
Removed.